garage/np/algos/base.py

```diff
@@ -5,3 +5,17 @@ class Algorithm:
 class RLAlgorithm(Algorithm):
     def train(self):
         raise NotImplementedError
+
+    def process_samples(self, itr, paths):
+        """Return processed sample data (typically a dictionary of concatenated
+        tensors) based on the collected paths.
+
+        Args:
+        itr (int): Iteration number.
+        paths (list): A list of collected paths.
```

Comment:
what is the type inside the list? is it `list[dict]` or `list[str]` or `list[list]`, etc?